pom.xml

```diff
@@ -375,17 +375,33 @@
         </executions>
       </plugin>
 
-      <!-- Find Bugs -->
+      <!-- FindBugs -->
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>findbugs-maven-plugin</artifactId>
-        <version>2.5.2</version>
+        <version>3.0.2</version>
         <configuration>
           <excludeFilterFile>${findbugs.path}findbugs-exclude.xml</excludeFilterFile>
+          <includeFilterFile>${findbugs.path}findbugs-include.xml</includeFilterFile>
+          <!--
+              Enables analysis which takes more memory but finds more bugs.
+              If you run out of memory, changes the value of the effort element
```

Comment:
TODO: we should have a more detailed "how to build Tachyon" web page, covering different caveats